com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -334,7 +340,11 @@ private static void copyEngineBinary(Project project, Platform architecture, Fil
         // possibly also strip it
         final boolean strip_executable = project.hasOption("strip-executable");
         if (strip_executable) {
-            String stripTool = Bob.getExe(Platform.getHostPlatform(), stripToolName);
```

Comment:
I forgot the other platforms.
If for windows I can replace tool too, but for linux it requires extra dependencies on cpp lib, so I keep it as is for now. 

That's why we need this extra logic to pick right tool name depends on a platform.